src/main/webapp/i18n/en/complaint.json

```diff
@@ -47,10 +47,10 @@
         },
         "moreFeedback": {
             "title": "More Feedback Request",
-            "description": "If something is not clear enough, or you need more explanations, you can request the tutor to provide more feedback",
-            "descriptionTeam": "If something is not clear enough, or you need more explanations, you can request your tutor to provide more feedback",
-            "beDescriptive": "Please describe in detail what is unclear for you in the assessment. Your request will then be evaluated by the same reviewer.",
-            "beDescriptiveTeam": "Please describe in detail what is unclear for you in the assessment. Your tutor will then clarify their assessment.",
+            "description": "If something is not clear enough and you need more explanation, you can request the tutor to provide more feedback",
+            "descriptionTeam": "If something is not clear enough and you need more explanation, you can request your tutor to provide more feedback",
```

Comment:
See above. If it turns out that every instance of a period missing from the end of a sentence is a mistake, then I'll gladly add them in. Otherwise I have no idea which strings should and should not have a period at the end.

Reply:
I also don't want to put all the burden on you of changing to more consistent punctuation and capitalization. As this PR is about translation, we should maybe focus on the language itself. @krusche Do you see the value in putting all that effort into revising punctuation and capitalization now?

Reply:
I think we have more important issues than punctuation and capitalization, but if there are volunteers, I also would not object against it :)